### ISSUE TITLE
[midiin] and sysex messages

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#285]
we were discussing this in https://github.com/pure-data/pure-data/pull/263#event-1412852064

A new help file in 0.48-1 now says [midiin] doesn't send sysex messages, but this seems to be wrong. I need a confirmation wether we want it to send sysex messages or not before we can fix this.

**Comment by danomatika** [comment]
Yes, [midiin] also sends sysex bytes.

That was the behavior before and I confused myself by accidentally changing it, then changing it back which also confused your help patch update.

**Comment by porres** [comment]
Ok, now I have a new PR which corrects this and says [midiin] takes ALL raw MIDI except real-time messages only, as it was before.
